app/src/androidTest/kotlin/org/wordpress/aztec/demo/tests/MixedTextFormattingTests.kt

```diff
@@ -107,4 +107,23 @@ class MixedTextFormattingTests : BaseTest() {
                 .verifyHTML(text)
     }
 
+    // Test reproducing the issue described in
+    // https://github.com/wordpress-mobile/AztecEditor-Android/pull/476#issuecomment-327762497
+    @Test
+    fun testMixedBoldItalics() {
```

Comment:
Ah right I didn't notice this, but this is strange: `testBoldAndItalicFormatting` passes on all device I tested, and the expected HTML tags are always in that order `<b><i>$text3</i></b>` (italics _inside_ bold). That's not the case for `testMixedBoldItalics`, on API 16 I get `<i><b>c </b></i>` and API > 16 I get `<b><i>c </i></b>` (and that's why I used a Regex here).

Reply:
Hmmm, odd. I consistently get `<b><i>c </i></b>` even on API 16. Since I can't reproduce the issue on my end I'm not sure what's going on, but I wonder if that's related to the separate spaces/the keyboard being dismissed between the calls to `insertText`? That's the only significant difference I see in the test steps.

Reply:
Dropped this test and moved the comment in 1ff934b